CMakeLists.txt

```diff
@@ -34,6 +34,7 @@ option(WITH_MAN "Enables hawkey man page generation" ON)
 option(WITH_ZCHUNK "Build with zchunk support" ON)
 option(ENABLE_RHSM_SUPPORT "Build with Red Hat Subscription Manager support?" OFF)
 option(ENABLE_SOLV_URPMREORDER "Build with support for URPM-like solution reordering?" OFF)
+option(ENABLE_SELINUX "Restore SELinux labels on GnuPG directories" ON)
```

Comment:
May I ask you why you want to make this option optional for compilation?

Reply:
This is a downstream patch, it is ON by default and it is not conditional in spec.

Reply:
It is conditional in spec (%bcond_without selinux).
I used the same approach as in librepo. I can make it unconditional if you want. But I would rather prefer uniformity.

Reply:
Please don't block the PR on my question. I am only curious what is behind. It is good approach to have it similar to `librepo`. Anyway, what was the reason to have it conditional in librepo? It might be due to adding a new dependency, potential issues, personal preferences or the condition is related to flag of `librepo` crypto backend. I am asking because I am curious.

Reply:
I made the dependency optional because librepo works without SELinux. For distributions that do not use SELinux the new dependency would be quite pointless.